### ISSUE TITLE
[9.*] Project GANTT showing HTML code in mousover

### ISSUE THREAD

**Issue by finalbeta** [glpi-project/glpi#8445]
**Describe the bug**

When using GANTT in projects the mousover showing the task description shows HTML code as text. The HTML code should probably be filtered out. 

**Page(s) URL**

front/project.form.php

**Version**
9.5.3

**To reproduce**

1. Create a project. 
2. Create a task in the project with a description that has numbers or list items and give expected start/end. 
3. Go to gant and hover over the task. :
![image](https://user-images.githubusercontent.com/22134261/102982711-446ec900-450b-11eb-9c27-6a34cebd844b.png)

**Expected behavior**

I understand HTML is not rendered. I think it should be filtered out in the mouseover though.

**Comment by finalbeta** [comment]
Side note. When I look at the description for the same task on the project task view, it displays in HTML mode. Looks like GANTT is using a differend mousover.
![image](https://user-images.githubusercontent.com/22134261/102985362-57839800-450f-11eb-8711-39bc9aa1b09e.png)

**Comment by cedric-anne** [comment]
Hi,

Gantt view has been completely refactored in master branch. This bug will not be fixed in 9.5 branch.

Regards

**Comment by derry43** [comment]
I have done a quick fix for this on 9.5.7 by modifying the function **getDataToDisplayOnGantt** in `inc/project.class.php` and changing the following lines 1806-1814 from:

```
         $todisplay[$real_begin.'#'.$real_end.'#project'.$project->getID()]
                      = ['id'       => $project->getID(),
                              'name'     => $project->fields['name'],
                              'link'     => $project->getLink(),
                              'desc'     => $project->fields['content'],
                              'percent'  => isset($project->fields['percent_done'])?$project->fields['percent_done']:0,
                              'type'     => 'project',
                              'from'     => $real_begin,
                              'to'       => $real_end];
```
to
```
         $todisplay[$real_begin.'#'.$real_end.'#project'.$project->getID()]
                      = ['id'       => $project->getID(),
                              'name'     => $project->fields['name'],
                              'link'     => $project->getLink(),
                              'desc'     => HTML::clean($project->fields['content']),
                              'percent'  => isset($project->fields['percent_done'])?$project->fields['percent_done']:0,
                              'type'     => 'project',
                              'from'     => $real_begin,
                              'to'       => $real_end];

```

and modifying the function **getDataToDisplayOnGantt** in `inc/projecttask.class.php` and changing the following lines 1571-1581 from:

```
        $todisplay[$real_begin.'#'.$real_end.'#task'.$task->getID()]
                        = ['id'    => $task->getID(),
                              'name'    => $task->fields['name'],
                              'desc'    => $task->fields['content'],
                              'link'    => $task->getlink(),
                              'type'    => 'task',
                              'percent' => $percent,
                              'from'    => $real_begin,
                              'parents' => $parents,
                              'to'      => $real_end,
                              'is_milestone' => $task->fields['is_milestone']];
```
to

```
        $todisplay[$real_begin.'#'.$real_end.'#task'.$task->getID()]
                        = ['id'    => $task->getID(),
                              'name'    => $task->fields['name'],
                              'desc'    => HTML::clean($task->fields['content']),
                              'link'    => $task->getlink(),
                              'type'    => 'task',
                              'percent' => $percent,
                              'from'    => $real_begin,
                              'parents' => $parents,
                              'to'      => $real_end,
                              'is_milestone' => $task->fields['is_milestone']];

```
